marshal/cluster.go

```diff
@@ -41,6 +41,8 @@ type KafkaCluster struct {
 	marshalers []*Marshaler
 	topics     map[string]int
 	groups     map[string]map[string]*topicState
+	// pausedGroups stores the expiry time for groups that are paused.
+	pausedGroups map[string]time.Time
```

Comment:
For future: I would have made `groups` which is the state of all groups change, instead be like,

```
groups map[string]groupState

type groupState struct {
    pauseTime time.Time
    topicStates map[string]*topicState
}
```

I don't think we need to do this now I just want to note it.